contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -44,7 +44,7 @@ strip /usr/local/clang+llvm/bin/* && \
 #
 # iproute2
 #
-git clone -b v4.10.0 git://git.kernel.org/pub/scm/linux/kernel/git/shemminger/iproute2.git && \
+git clone -b v4.12.0 git://git.kernel.org/pub/scm/network/iproute2/iproute2.git && \
```

Comment:
Any concern with us updating as crucial a dependency as iproute2 ? I'm wondering if we should do additional testing with these new containers to ensure we won't get bitten by iproute2 bugs. My worry could be unwarranted, though.

Reply:
@borkmann said it was ok, also I was not able to build the image any longer for 16.04 and this can be for `1.1`

Reply:
Great, thanks!